ludwig/config_validation/checks.py

```diff
@@ -472,6 +472,22 @@ def check_llm_atleast_one_input_text_feature(config: "ModelConfig"):  # noqa: F8
     raise ConfigValidationError("LLM requires at least one text input feature.")
 
 
+@register_config_check
+def check_llm_finetuning_output_feature_config(config: "ModelConfig"):  # noqa: F821
+    """Checks that the output feature config for LLM finetuning is valid."""
+    if config.model_type != MODEL_LLM:
+        return
+
+    if config.trainer.type != "finetune":
+        return
+
+    if config.output_features[0].type != TEXT:
```

Comment:
Couple sanity checks here:
-  We only allow one output feature for LLMs currently right?
- I assume there's another validation check for this, but should we check to make sure `config.output_features` is not an empty list?

Reply:
> We only allow one output feature for LLMs currently right?

Technically yes, but I actually don't think we have any explicit checks for this today. Let me add another one for this particular scenario in another PR.

> I assume there's another validation check for this, but should we check to make sure config.output_features is not an empty list?

I would hope that we have a check for this already, or that normal config validation through marshmallow would fail if we got an empty list for output features. I'm quite sure that is already the case so wouldn't worry about this one too much.

Reply:
Okay I just tried an LLM model type with 1 input feature and 2 output features and this is the error I got:

```
E           ludwig.error.ConfigValidationError: Failed to validate JSON schema for config. Error: [{'name': 'text_output', 'type': 'text'}, {'name': 'category_output', 'type': 'category'}] is too long
```

I attribute this to this check in the LLm model type schema: https://github.com/ludwig-ai/ludwig/blob/master/ludwig/schema/features/base.py#L320

So seems like we don't have to take care of this, but I will say that the error message is non-obvious/quite cryptic.